configs/testing-library-compass/src/index.tsx

```diff
@@ -732,6 +732,17 @@ const testingLibrary = {
   renderHook,
 };
 
+/**
+ * Expects an element (queried test id) to visible or missing from the screen
+ */
+function expectElementByTestId(testId: string, state: 'visible' | 'missing') {
+  if (state === 'visible') {
+    expect(screen.queryByTestId(testId)).to.be.visible;
+  } else if (state === 'missing') {
+    expect(screen.queryByTestId(testId)).to.be.null;
+  }
+}
```

Comment:
If we want a generalized helper I think it would make a bit more sense so add this as an extra chai assertion instead of a function here, it's more effort but way more consistent with how we assert, which is always done with chai library asserts

Reply:
I did experiment a bit with it. The question is what we'd want that `expect` to take as an argument.
If it takes the `test-id` string attribute, I don't know what to call that chai assert property 🙈 `expect("some-test-id").to.be.testIdOfElementInDocument`?

Thinking more about it, perhaps it's overkill with this helper 🤔 I mostly wanted to find a way to codify the pattern.

Reply:
I've reverted the helper and cleaned up the test instead: 4450dc71faf479e74eb93675e30cf0e9d3c5c830 - I think that's better 🤞